android/filament-android/src/main/java/com/google/android/filament/Renderer.java

```diff
@@ -356,6 +356,48 @@ public void render(@NonNull View view) {
         nRender(getNativeObject(), view.getNativeObject());
     }
 
+   /**
+     * Renders a standalone {@link View} into its associated <code>RenderTarget</code>.
+     *
+     * <p>
+     * The purpose of the <code>renderStandaloneView()</code> function is to generate render
+       commands which are asynchronously executed by the {@link Engine}'s render thread.
```

Comment:
This is somewhat cryptic, end users don't know what "render commands" are.  Maybe we should instead say that this can be used for doing work that is not associated with any particular swap chain?  Also I think a `*` is missing.

Reply:
This is just a copy of the language for render() -- which is also kind of cryptic :-)

Reply:
Okay I tried to remove the confusing language.